lib/stubify/server.rb

```diff
@@ -15,28 +15,31 @@ class Server < Sinatra::Base
 
     at_exit { Server.run! if $!.nil? && Server.run? }
 
-    get // do
+    get '/*' do
+      Server.log("Processing request GET #{request}")
       response = Server.on_request(request)
       status response['status_code']
       return response[:body] unless response[:body].nil?
       return response['body'] unless response['body'].nil?
     end
 
-    post // do
+    post '/*' do
+      Server.log("Processing request POST #{request}")
```

Comment:
are these still required?

Reply:
I think that makes sense but in that case I would make it consistent and add logs to the other http verbs too :)

Reply:
👍